Editor/Exporter.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.IO;
+
+namespace Unity.ProjectAuditor.Editor
+{
+    public abstract class Exporter : IDisposable
+    {
+        protected readonly IssueLayout m_Layout;
+        protected readonly StreamWriter m_StreamWriter;
+
+        public Exporter(string path, IssueLayout layout)
+        {
+            m_Layout = layout;
+            m_StreamWriter = new StreamWriter(path);
+        }
+
+        public void Dispose()
+        {
+            m_StreamWriter.Flush();
+            m_StreamWriter.Close();
+        }
+
+        public virtual void WriteHeader() { }
```

Comment:
can we make this _abstract_ since we always need an overload ?

Reply:
Good point. I did not realize it.